package.json

```diff
@@ -73,6 +74,7 @@
     "react-hot-loader": "^4.12.11",
     "redux-testkit": "^1.0.6",
     "shader-loader": "^1.3.1",
+    "start-server-and-test": "^1.9.1",
```

Comment:
I don't think this is needed now that we're using the Cypress orb. See `wait-on` property